src/components/scenes/TransactionDetailsScene.js

```diff
@@ -16,7 +16,7 @@ import * as Constants from '../../constants/indexConstants'
 import { formatNumber } from '../../locales/intl.js'
 import s from '../../locales/strings.js'
 import { getDisplayDenomination, getPlugins, getSettings } from '../../modules/Settings/selectors.js'
-import { convertCurrencyFromExchangeRates, convertNativeToExchangeRateDenomination, getSelectedWallet, getWallet } from '../../modules/UI/selectors.js'
+import { convertCurrencyFromExchangeRates, convertNativeToExchangeRateDenomination } from '../../modules/UI/selectors.js'
```

Comment:
should this methods also be in some kind of util file?

Reply:
I agree that these don't seem like normal selectors, but my goal in this PR isn't to organize everything - just get rid of selectors.

Reply:
ok